doc/attr.xml

```diff
@@ -1211,3 +1211,60 @@ gap> OutNeighbours(sym);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="UndirectedSpanningTree">
+<ManSection>
+  <Attr Name="UndirectedSpanningTree" Arg="digraph"/>
+  <Attr Name="UndirectedSpanningForest" Arg="digraph"/>
+  <Returns>A digraph, or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph with at least one vertex, then
+    <C>UndirectedSpanningForest</C> returns an undirected spanning forest of
+    <A>digraph</A>, otherwise this attribute returns <K>fail</K>. See <Ref
+      Oper="IsUndirectedSpanningForest" /> for the definition of an undirected
+    spanning forest.
+
```

Comment:
The way I think of it, the `<P/>` tags marks the boundary between paragraphs. So I like the style of putting it clearly between the preceding and proceeding paragraphs. But I'll happily change it.